include/map_object.h

```diff
@@ -1,11 +1,11 @@
-#ifndef POKEHEARTGOLD_FIELD_MAP_OBJECT_H
-#define POKEHEARTGOLD_FIELD_MAP_OBJECT_H
+#ifndef POKEHEARTGOLD_MAP_OBJECT_H
+#define POKEHEARTGOLD_MAP_OBJECT_H
 
 #include "script.h"
 
 typedef struct SavedMapObject {
-    u32 unk0;
-    u32 unk4;
+    u32 flags;
+    u32 flags2;
```

Comment:
any ay to distinguish between flags and flags2?

Reply:
honestly I forgot I even renamed these in this ptr, I'll check when I have more time lmao

Reply:
flags and flags 2 are copied from the saved objects, since they correspond, as such atm it's not exactly known what these are